scripts/acceptance.sh

```diff
@@ -2,14 +2,13 @@
 
 set -ex
 
-if [ ! -f server.dist.js ]; then
-  # Build the client assets for testing
-  export BUILD_CLIENT=true
+export NODE_ENV=production
 
-  # CSS is only compiled during a development build?
-  export NODE_ENV=production
+if [ ! -f server.dist.js ]; then
+  yarn build:assets
+  yarn build:assets:novo
 
-  yarn assets
+  # TODO: We don't need the server to be built here, these tests only rely on assets.
```

Comment:
Is this comment still accurate? Not sure why we build server when they rely on assets.